license

```diff
@@ -0,0 +1,21 @@
+The MIT License
```

Comment:
I looked into this.  I recommend using the MIT license: https://opensource.org/licenses/MIT

Make it `Copyright (c) 2015-2016 Trung Le, Shuai Shao (Shrek), and Contributors`

I also made a CLA for contributors to sign (including me!) based on the Linux DCO: https://gist.github.com/pjcozzi/4d1ab2166519de7ba41b

And I set this repo up with https://cla-assistant.io/ to handle the CLAs (this is the first time I am using it).